src/pip/_internal/network/xmlrpc.py

```diff
@@ -0,0 +1,36 @@
+import logging
```

Comment:
A documentation string describing the contents of this module would be nice to have! ^>^